scss/challengeTitleCard.scss

```diff
@@ -0,0 +1,8 @@
+.challenge-title-card--done {
+    color: #6d7278;
+    background-color: rgba(255, 255, 255, 0.5);
+}
+
+.challenge-title-card--current {
+    border-left: 3px solid #5438dc;
```

Comment:
for colors, could we use variables?

Reply:
Ah yes, been ages since I used scss. Just saw the index.scss file with variables :)

Reply:
https://github.com/garageScript/c0d3-app/blob/master/scss/index.scss

Reply:
Fixed :)